winlogbeat/checkpoint/file_windows.go

```diff
@@ -29,8 +30,8 @@ func createWriteThroughFile(path string) (*os.File, error) {
 		syscall.GENERIC_READ|syscall.GENERIC_WRITE,               // Access Mode
 		uint32(syscall.FILE_SHARE_READ|syscall.FILE_SHARE_WRITE), // Share Mode
 		nil, // Security Attributes
-		syscall.CREATE_ALWAYS,                                          // Create Mode
-		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH), // Flags and Attributes
+		syscall.CREATE_ALWAYS, // Create Mode
+		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH|_FILE_FLAG_NO_BUFFERING), // Flags and Attributes
```

Comment:
@andrewkroh I tried to find O_SYNC for windows and it seems to be the NO_BUFFERING. Looking into filebeat, we do not treat creating the registry for different OS differently. Should we?

In any case, we should still unify registrar handling in libbeat.